src/components/MemoList.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="memo-list-area">
+    <div class="memo-list-item" v-for="memo in memos" :key="memo.id">
+      <router-link :to="`/memos/${memo.id}`">{{ getMemoTitle(memo.content) }}</router-link>
```

Comment:
```suggestion
      <router-link :to="`/memos/${memo.id}`">{{ getMemoTitle(memo) }}</router-link>
```

としてもいいかも。

このままにするならメソッド定義側を`getMemoTitle(content) {`としたいですね